frontend/src/libs/packages/store/middlewares/handle-error.middleware.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  type AnyAction,
+  type Dispatch,
+  type Middleware,
+} from '@reduxjs/toolkit';
+import { isRejected } from '@reduxjs/toolkit';
+
+import { NotificationService } from '#libs/helpers/notification-service.js';
+
+const handleAction =
+  (next: Dispatch): ((action: AnyAction) => AnyAction) =>
+  (action: AnyAction): AnyAction => {
+    if (isRejected(action) && action.error.message) {
+      NotificationService.error(action.error.message);
```

Comment:
Use dispatch to call action that contains NotificationService.error(action.error.message);